src/Storage/Compression/Algorithms/Zopfli.php

```diff
@@ -0,0 +1,41 @@
+@@ -0,0 +1,40 @@
+<?php
+
+namespace Utopia\Storage\Compression\Algorithms;
+
+use Utopia\Storage\Compression\Compression;
+
+class Zopfli extends Compression
+{
+    /**
+     * @return string
+     */
+    public function getName(): string
+    {
+        return 'zopfli';
+    }
+
+    /**
+     * Compress.
+     *
+     * @param string $data
+     *
+     * @return string
+     */
+    public function compress(string $data): string
+    {
+        return \zopfliencode($data);
+    }
+
+    /**
+     * Decompress.
+     *
+     * @param string $data
+     *
+     * @return string
+     */
+    public function decompress(string $data): string
+    {
+        return \zopflidecode($data);
```

Comment:
These are not the correct encode/decode functions for the new extension repo.

Reply:
changing this one!

Reply:
This hasn't been updated.

Reply:
Updated now